src/mapping/tests/MappingConfigurationTest.cpp

```diff
@@ -47,7 +47,7 @@ BOOST_AUTO_TEST_CASE(Configuration)
 
 BOOST_AUTO_TEST_CASE(RBFDirectConfiguration)
 {
-  PRECICE_TEST(1_rank);
+  PRECICE_TEST(1_rank, Require::PETSc);
```

Comment:
I don't see where this configuration test requires PETSc. If it is required for practical purposes, please add documentation here why it should be required.

Reply:
Good catch! This slipped into the commit.

FYI I added PETSc to the alias tag test, as this may choose PETSc in the future.